src/runtime_src/core/common/api/xrt_xclbin.cpp

```diff
@@ -962,6 +996,12 @@ get_axlf_section(const xrt::xclbin& xclbin, axlf_section_kind kind)
   return xclbin.get_handle()->get_axlf_section(kind);
 }
 
+std::vector<std::pair<const char*, size_t>>
```

Comment:
Yes, this is format to use, can you fix the other instances like on line 622 to be same as this?